### PR TITLE
Fix devices error

### PR DESCRIPTION
**Pull request by karaggeorge**
Fixes Sentry error [KAP-3G8](https://sentry.io/organizations/wulkano-l0/issues/832030699/?project=255536&query=is%3Aunresolved&statsPeriod=7d)

`TypeError: devices.some is not a function`

### CODE REVIEW

**Comment by sindresorhus** [comment]
I agree with the change, but it would be good to figure out why `devices` is not an array in the first place.

File: main/common/settings.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ const audioInputDeviceId = store.get('audioInputDeviceId');
 (async () => {
   const devices = await audioDevices();
```
Comment (sindresorhus) [inline main/common/settings.js]: Note https://github.com/wulkano/aperture-node/blob/ef5cd98b9cc6496405bf36177dea79a11cf4952f/index.js#L165, so we could maybe do:

```js
if (!Array.isArray(devices)) {
	// Sentry logging here
	// console.log(devices);

	return
}
```

File: main/common/settings.js [hunk 2]
```diff
@@ -90,12 +92,16 @@ const audioInputDeviceId = store.get('audioInputDeviceId');
...
 
+  if (!Array.isArray(devices)) {
+    const Sentry = require('../utils/sentry');
+    Sentry.captureMessage(`devices is not a string: ${JSON.stringify(devices)}`);
```
Comment (sindresorhus) [inline main/common/settings.js]: You mean not an array?

```suggestion
    Sentry.captureMessage(`devices is not an array: ${JSON.stringify(devices)}`);
```

Comment (karaggeorge) [reply main/common/settings.js]: 🤦‍♂ Yes

File: main/common/settings.js [hunk 3]
```diff
@@ -90,12 +92,16 @@ const audioInputDeviceId = store.get('audioInputDeviceId');
 (async () => {
   const devices = await audioDevices();
 
+  if (!Array.isArray(devices)) {
+    const Sentry = require('../utils/sentry');
+    Sentry.captureMessage(`devices is not an array: ${JSON.stringify(devices)}`);
```
Comment (sindresorhus) [inline main/common/settings.js]: It's unclear from the [docs](https://github.com/getsentry/sentry-electron) if Sentry saves the message here when there's no exception. I think `captureMessage` is meant to add additional context to an error, but there's no error here. Let's use `captureException` just to be sure it's saved.

Comment (sindresorhus) [reply main/common/settings.js]: // @karaggeorge  ⬆️

Comment (karaggeorge) [reply main/common/settings.js]: Sorry, have been super busy at work lately 😅. Used `captureException`, their types say that it accepts `any` but the annotation says 

> an exception-like object

So, I passed in a new error. That should work right?

Comment (sindresorhus) [reply main/common/settings.js]: Yup, I would assume so.

**Pull request merged**